### PR TITLE
ia-writer-quattro: init at unstable-2023-06-16

### PR DESCRIPTION
**Pull request by x0ba**
## Description of changes

This commit adds the [iA Writer Quattro](https://github.com/iaolo/iA-Fonts) typeface and adds myself as a maintainer.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by katexochen** [comment]
Retry of https://github.com/NixOS/nixpkgs/pull/268647

File: maintainers/maintainer-list.nix
Comment (katexochen) [inline maintainers/maintainer-list.nix:1]: I think you accidentally formatted this file.

File: pkgs/by-name/ia/ia-writer-quattro/package.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+,
```
Comment (katexochen) [inline pkgs/by-name/ia/ia-writer-quattro/package.nix:4]:
```suggestion
```

**Review by katexochen — CHANGES_REQUESTED** [review]
Your PR should have exactly 2 commits:

- `maintainers: add x0ba`
- `ia-writer-quattro: init at unstable-2023-06-16`

**Review by Lurkki14 — APPROVED** [review]

**Review by katexochen — APPROVED** [review]

**Pull request merged**